src/js/components/LanguageWrapper.js

```diff
@@ -19,6 +20,12 @@ class LanguageWrapper extends Component {
     }
   }
 
+  getChildContext() {
```

Comment:
I didn't know much about this React method. Seems like a good use case for what we're trying to do here! I definitely like it better than the React-Static specific HOC pattern.

I found this article that also lists out a few things to be careful of with React context. Many just gotchas around debugging and (over)coupling together two components.
https://javascriptplayground.com/context-in-reactjs-applications/

If you have any good resources you read that gave you the idea to implement this pattern, please share, I'd love to learn more.

Reply:
for sure, I've mostly been looking at [React docs](https://reactjs.org/docs/context.html) straight up and how Links and Routers work in react-router-v4 -- since Links were super nested all over and need to have Routers in scope, I was just looking at how that was handled.  Thanks for the info link -- I definitely agree that context can complicate things when overused, but here it seems like the right solution, since Links, and maybe other components, might need to be aware of langCode for translation.